### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by AbsurdUsername**

### CODE REVIEW

File: README_IT.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#### [EN](https://github.com/sakusaku3939/DeepLAndroid#readme) | [JA](https://github.com/sakusaku3939/DeepLAndroid/blob/master/README_JA.md) | [RU](https://github.com/sakusaku3939/DeepLAndroid/blob/master/README_RU.md) | [DE](https://github.com/sakusaku3939/DeepLAndroid/blob/master/README_DE.md) | [FR](https://github.com/sakusaku3939/DeepLAndroid/blob/master/README_FR.md) | [IT]https://github.com/sakusaku3939/DeepLAndroid/blob/master/README_IT.md)
```
Comment (sakusaku3939) [inline README_IT.md:1]: There is no `(` next to `[IT]`.

**Review by sakusaku3939 — COMMENTED** [review]
Thanks for adding the translations!
I found a simple mistake.

File: README_IT.md (hunk 1 above)
Comment (AbsurdUsername) [reply README_IT.md]: Ops sorry, thank you for pointing it out, fixed.

**Comment by sakusaku3939** [comment]
It seems to be no problem.
Thank you!

**Pull request merged**